enhancements/template.md

```diff
@@ -193,6 +193,24 @@ enhancement:
 - What changes (in invocations, configurations, API use, etc.) is an existing
   cluster required to make on upgrade in order to make use of the enhancement?
 
+Upgrade expectations:
+- Micro version upgrades - users should be able to skip forward versions within a
+  minor release stream without being required to pass through intermediate
+  versions - i.e. `x.y.N->x.y.N+2` should work without requiring `x.y.N->x.y.N+1`
+  as an intermediate step.
+- Minor version upgrades - you only need to support `x.N->x.N+1` upgrade
+  steps. So, for example, it is acceptable to require a user running 4.3 to
+  upgrade to 4.5 with a `4.3->4.4` step followed by a `4.4->4.5` step.
```

Comment:
I would add a couple here:

1. Every component is expected to remain available for user requests and workloads during upgrades
2. Brief disruption is only allowed by special exception
3. An upgrade is a series of subsystem and component upgrades - every component MUST tolerate both Z and Y level (4.y.z) differences in their componetns for arbitrary amounts of time in the event of a disruption to the system during upgrade.  I.e. if a node is down, and an operator is rolling out a daemonset, the old and new daemonset pods must continue to work without issue during that upgrade against whatever components have been upgraded already and whatever components have not yet been upgrade.

Reply:
Thanks, added